### PR TITLE
Fixes #150 Add new descriptions for cats

### LINKED ISSUE #150: Better cat Data
Whoever is going to work on the tasks gets the privilege to chose the best cats (or maybe even switch to puppies. Go puppy tube!)

### PR DESCRIPTION
**Pull request by MichaelSolati**
Add new cats as well

### CODE REVIEW

**Comment by MichaelSolati** [comment]
bitmoji

**Comment by joeycatxx** [comment]
cat 1
cat 2
cat 3
cat 4
cat 5
cat 6
cat 7
cat 8
cat 9
cat 10

File: ng2ts/api.service.ts [hunk 1]
```diff
@@ -2,75 +2,163 @@ import {VideoItem} from './video/video-item';
...
+    src: '/assets/images/cat-01.jpg',
+    description: 'The Itty Bitty Kitty Comittee are a collection of felines coming for you!',
     views: 100,
     likes: 20,
     date: '2016-11-21'
   }, {
-    title: 'More kitten',
-    src: '/assets/images/cat-2.jpg',
-    description: 'todo',
+    title: 'IT A KITTY!',
```
Comment (prestonvanloon) [inline ng2ts/api.service.ts]: typo(?) `IT'S A KITTY!`

File: ng2ts/api.service.ts [hunk 2]
```diff
@@ -2,75 +2,163 @@ import {VideoItem} from './video/video-item';
...
+  {
+    title: 'Nikki\'s First Feast',
+    src: '/assets/images/cat-14.jpg',
+    description: 'Nikki tries fancy cat food.',
+    views: 99,
+    likes: 1,
+    date: '2016-09-23'
+  },
+  {
+    title: 'Thunder Scared Of Lightening',
```
Comment (prestonvanloon) [inline ng2ts/api.service.ts]: `Lightning`

File: ng2ts/api.service.ts (hunk 1 above)
Comment (MichaelSolati) [inline ng2ts/api.service.ts]: Maybe? Pseudo [lolcat speak](http://speaklolcat.com/)

Comment (prestonvanloon) [inline ng2ts/api.service.ts]: `¯\_(ツ)_/¯`

**Comment by prestonvanloon** [comment]
LGTM

**Pull request merged**